lib/common.js

```diff
@@ -294,7 +294,17 @@ exports.createTableForEnvironments = function (environments) {
       targets.push(target._id);
     });
 
-    table.push([env._id, env.label, env.autoDeployOnCreate, env.autoDeployOnUpdate, targets.join(", "), moment(env.modified).fromNow()]);
+    // Display undefined options as 'false'
+    var autoDeployOnCreate = env.autoDeployOnCreate
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)